examples/example-invoice-csv-export/src/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1",
+  "information": {
+    "name": "example-invoice-csv-export",
+    "displayName": "[Example] Invoice CSV export",
+    "description": "This is a sample UCRM plugin showcasing a custom link in menu and security checks.",
+    "url": "https://github.com/Ubiquiti-App/UCRM-plugins",
```

Comment:
link directly into plugin's directory, so that plugin's readme is displayed

Reply:
```suggestion
    "url": "https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/example-invoice-csv-export",
```